src/commands/admin/resetlevels.ts

```diff
@@ -43,14 +46,17 @@ export default class extends Command {
 			}
 		});
 
-		msg.client.mongo.collection(DB.USERS).updateMany({}, { $set: {
-			count: 0,
-			levelExp: FIRST_LEVEL,
-			level: 1,
-			curExp: FIRST_LEVEL
-		} });
+		msg.client.mongo.collection<SageUser>(DB.USERS).updateMany(
+			{ roles: { $all: [ROLES.VERIFIED] } }, {
+				$set: {
+					count: 0,
+					levelExp: FIRST_LEVEL,
+					level: 1,
+					curExp: FIRST_LEVEL
+				}
+			});
```

Comment:
r.i.p. indent :(

Serious note: Has it been decided to leave everyone at level 1 upon resetting? I'm not too sure if my memory serves me wrong but I thought we'd set everyone's level to 0 (unless that has changed as we are resetting levels right when next sem starts).

Reply:
There is no level 0

Reply:
Oh, of course. :P